client/js/lounge.js

```diff
@@ -447,8 +425,49 @@ $(function() {
 			socket.emit("sessions:get");
 		}
 
+		if (target === "#help" || target === "#changelog") {
```

Comment:
` || target === "#changelog"` because if we're loading the page while on the changelog, we need to retrieve its content. Now, well, this isn't working because of something entirely different (handlers are not set up so we're sent to a lobby), but it's ready to support it.